acceptancetests/assess_deploy_webscale.py

```diff
@@ -263,7 +271,7 @@ def mongo_snap_settings(args):
 
     # Fetch snap if using a remote URL. Juju expects the snap file to match
     # "juju-db_\d+.snap" so we should rename it accordingly.
-    dst_snap_path = "/tmp/juju-db_1.snap"
+    dst_snap_path = "/tmp/juju-db_6.snap"
```

Comment:
why is this now 6 vs 1?

Reply:
No particular reason. I fetched the actual snap we are using internally from the edge channel and it has the `_6` suffix so I just copied that over; I think it doesn't really matter (AFAICT the asserts file does not include a reference to the snap version? suffix)

Reply:
Ok, fair enough. So it's not something we have to continuously edit/keep up to date or the like?